### PR TITLE
added knobs to btn

### PR DESCRIPTION
**Pull request by kachakot**

### CODE REVIEW

File: src/components/ButtonLink/ButtonLink.stories.js [hunk 1]
```diff
@@ -1,31 +1,28 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
+import { text, select, boolean } from '@storybook/addon-knobs';
```
Comment (rgolubowicz) [inline src/components/ButtonLink/ButtonLink.stories.js]: remove semicolon

File: src/components/ButtonLink/ButtonLink.stories.js [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
 
 import ButtonLink, { types, sizes } from './ButtonLink'
 
 storiesOf('ButtonLink', module)
-  .add('primary medium', () => (
+.add('default', () => {
```
Comment (rgolubowicz) [inline src/components/ButtonLink/ButtonLink.stories.js]: add indents

File: src/components/ButtonLink/ButtonLink.stories.js (hunk 1 above)
Comment (kachakot) [reply src/components/ButtonLink/ButtonLink.stories.js]: done

File: src/components/ButtonLink/ButtonLink.stories.js (hunk 2 above)
Comment (kachakot) [reply src/components/ButtonLink/ButtonLink.stories.js]: done

**Pull request merged**